dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1024,6 +1030,40 @@ LanguageClientInterface convertCLIStringToEnum(String descriptor) {
         throw new UnsupportedOperationException("language not supported yet");
     }
 
+    /**
+     * Loads docker images from file system if there are any
+     */
+    public void loadDockerImages() {
+        INIConfiguration config = Utilities.parseConfig(this.getConfigFile());
+        String dockerImageDirectory = config.getString("docker-images");
+        if (!StringUtils.isBlank(dockerImageDirectory)) {
+            Path directoryPath = Paths.get(dockerImageDirectory);
+            Supplier<Stream<Path>> list = () -> {
+                try {
+                    return java.nio.file.Files.list(directoryPath);
+                } catch (NotDirectoryException e) {
+                    System.out.println("Directory is a file: " + directoryPath.toAbsolutePath());
```

Comment:
Say what you're looking for, this might be the first error they see and without context, they won't know what directory is a file